shared_libs/message_bus_lib/message_bus_lib/servicebus_client_factory.py

```diff
@@ -47,12 +47,14 @@ def create_message_receiver_client(
         self.logger.debug(
             "Creating message receiver client for queue '%s' with session_id '%s'", queue_name, session_id
         )
+
+        lock_renewal = AutoLockRenewer(max_lock_renewal_duration=MAX_LOCK_RENEWAL_DURATION) if session_id else None
```

Comment:
just to confirm that we want to create a new one for each receiver instance, I'm guessing since we're not using a pool of workers + our separate flows that this is the only option instead of just one to manage them